sway-book/src/documentation/language/functions/function.md

```diff
@@ -0,0 +1,19 @@
+# Functions
+
+In this section we will define a function that takes two numerical inputs and returns a [boolean](../built-ins/boolean.md) value indicating whether they are equal. We will also take a look at how to use the function.
+
+## Declaration
+
+The following function is called `equals` and it takes two parameters of type `u64` (64-bit unsigned integers). It performs a comparison and [implicitly](./return.md) returns the result of that comparison.
```

Comment:
```suggestion
The following function is named `equals` and it takes two parameters of type `u64` (64-bit unsigned integers). It performs a comparison and [implicitly](./return.md) returns the result of that comparison.
```

Reply:
Pretty sure names belong to people while things are `called` something